frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/delete-block-pool-modal.tsx

```diff
@@ -148,8 +152,8 @@ const DeleteBlockPoolModal = withHandlePromise((props: DeleteBlockPoolModalProps
 });
 
 type DeleteBlockPoolModalProps = {
-  kind?: string;
-  blockPoolConfig: StoragePoolKind;
+  kind?: K8sKind | string;
```

Comment:
```suggestion
  kind?: K8sKind;
```
This union is unholy.

Reply:
agreed, anyway, I am not using this kind. i can adjust this